app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :Username%><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "username" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] The views in devise for the registration and login should have been modified to match the wireframes and their styling.

=> You can take a look at the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.